mrobosub_hal/src/thruster_controller.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+import rospy
+
+from mrobosub_lib.lib import Node
+from std_msgs.msg import Float64
+from serial import Serial
+
+class ThrusterController(Node):
+    def __init__(self):
+        super().__init__('thruster_controller')
+        print("Launched thruster_controller node")
+        self.port = "/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_12-Channel_USB_Servo_Controller_00467345-if00"
+        # self.port = '/dev/ttyACM0'
+        self.get_errors() #just to clear errors at the start
+        
+        rospy.Subscriber('/motor_pwm/0', Float64, self.motor_0_callback)
+        rospy.Subscriber('/motor_pwm/1', Float64, self.motor_1_callback)
+        rospy.Subscriber('/motor_pwm/2', Float64, self.motor_2_callback)
+        rospy.Subscriber('/motor_pwm/3', Float64, self.motor_3_callback)
+        rospy.Subscriber('/motor_pwm/4', Float64, self.motor_4_callback)
+        rospy.Subscriber('/motor_pwm/5', Float64, self.motor_5_callback)
+        rospy.Subscriber('/motor_pwm/6', Float64, self.motor_6_callback)
+        rospy.Subscriber('/motor_pwm/7', Float64, self.motor_7_callback)
```

Comment:
We should decide what the topics should be called and coordinate with #57 to use the same names in both